mteb/tasks/Classification/tur/TurkishMovieSentimentClassification.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskClassification
+
+
+class TurkishMovieSentimentClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="TurkishMovieSentimentClassification",
+        description="Turkish Movie Review Dataset",
+        reference="https://www.win.tue.nl/~mpechen/publications/pubs/MT_WISDOM2013.pdf",
+        dataset={
+            "path": "asparius/Turkish-Movie-Review",
+            "revision": "409a4415cce5f6bcfca6d5f3ca3c408211ca00b3",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["tur-Latn"],
+        main_score="accuracy",
+        date=None,
+        form=None,
+        domains=None,
+        task_subtypes=None,
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators=None,
+        dialect=None,
+        text_creation=None,
```

Comment:
Please add this metadata. Feel free to take the best guess.

Reply:
From the paper it seems like the turkish data was collected. So this could be `"found"`.